rosidl_generator_rs/resource/srv.rs.em

```diff
@@ -0,0 +1,78 @@
+@{
+req_res_specs = []
+
+for subfolder, service in srv_specs:
+    req_res_specs.append((subfolder, service.request_message))
+    req_res_specs.append((subfolder, service.response_message))
+}@
+
+@{
+TEMPLATE(
+    'msg_idiomatic.rs.em',
+    package_name=package_name, interface_path=interface_path,
+    msg_specs=req_res_specs,
+    get_rs_name=get_rs_name, get_rmw_rs_type=get_rmw_rs_type,
+    get_idiomatic_rs_type=get_idiomatic_rs_type)
+}@
+
+@[for subfolder, srv_spec in srv_specs]
+
+@{
+type_name = srv_spec.namespaced_type.name
+}@
+
+#[link(name = "@(package_name)__rosidl_typesupport_c")]
+extern "C" {
+    fn rosidl_typesupport_c__get_service_type_support_handle__@(package_name)__@(subfolder)__@(type_name)() -> libc::uintptr_t;
+}
+
+pub struct @(type_name);
```

Comment:
Would be nice to add a little documentation here – for messages we have this line:
```
// Corresponds to @(package_name)__@(subfolder)__@(type_name)
```

Reply:
Done in 70f41be